type-definitions/immutable.js.flow

```diff
@@ -390,7 +400,7 @@ declare class SetSeq<T> extends Seq<T,T> mixins SetIterable<T> {
 }
 
 declare class List<T> extends IndexedCollection<T> {
-  static <T>(iterable?: ESIterable<T>): List<T>;
+  static (iterable?: ESIterable<T>): List<T>;
```

Comment:
Probably because I edited that constructor in my edits and just did not edit the other ones.
Although I ended up with exactly the same constructor I still rewrote it.
Gonna remove them everywhere.